### PR TITLE
Disaggregate vector species outputs:

### PR DESCRIPTION
**Pull request by giovannic**
* Disaggregate EIR and state counts by mosquito species
  * Remove redundant Total_M and and global EIR outputs
  * Update species output columns to use species names

### CODE REVIEW

File: R/biting_process.R [hunk 1]
```diff
@@ -118,6 +119,7 @@ simulate_bites <- function(
 
     lagged_eir[[s_i]]$save(n_infectious * a, timestep)
     species_eir <- lagged_eir[[s_i]]$get(timestep - parameters$de)
+    renderer$render(paste0('EIR_', species_name), species_eir, timestep)
     EIR <- EIR + species_eir
```
Comment (htopazian) [inline R/biting_process.R:123]: Great idea to disaggregate. Will it be easy for people to calculate overall EIR? Is this EIR the overall, or will people add(?) them?

Comment (giovannic) [reply R/biting_process.R:123]: Yup, total EIR is just the sum of each species EIR

**Review by htopazian — APPROVED** [review]
Great idea to diaggregate - anyone can do whatever they want with the outputs now!

**Comment by giovannic** [comment]
Not my idea, but I'll take all the credit :D

**Pull request merged**